vraptor-core/src/main/java/br/com/caelum/vraptor/serialization/gson/DateGsonConverter.java

```diff
@@ -41,25 +41,29 @@
 @Dependent
 public class DateGsonConverter implements JsonDeserializer<Date>, JsonSerializer<Date>{
 
-	private final SimpleDateFormat iso8601Format;
+	private final SimpleDateFormat format;
 
 	public DateGsonConverter() {
-		this.iso8601Format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
+		this.format = getDateFormat();
 	}
 	
 	@Override
 	public JsonElement serialize(Date date, Type typeOfSrc, JsonSerializationContext context) {
-		String dateString = iso8601Format.format(date);
+		String dateString = format.format(date);
 		return new JsonPrimitive(dateString);
 	}
 
 	@Override
 	public Date deserialize(JsonElement json, Type typeOfT, JsonDeserializationContext context) throws JsonParseException {
 		try {
-			return iso8601Format.parse(json.getAsString());
+			return format.parse(json.getAsString());
 		} catch (ParseException e) {
 			throw new JsonSyntaxException(json.getAsString(), e);
 		}
+	}	
+
+	protected SimpleDateFormat getDateFormat(){
```

Comment:
Change method return to `DateFormat`. This allow us to use any implementation of `DateFormat`, not only `SimpleDateFormat`.